### PR TITLE
fix: Video sender flips texture vertically by default

### PR DESCRIPTION
**Pull request by MaximKurbanov**
#981 
fix videoSender too

### CODE REVIEW

File: Runtime/Scripts/VideoStreamTrack.cs [hunk 1]
```diff
@@ -157,10 +157,7 @@ public VideoStreamTrack(Texture texture, CopyTexture copyTexture = null)
             var dest = CreateRenderTexture(texture.width, texture.height);
 
             m_source = source;
-            if (copyTexture != null)
-            {
-                m_source.copyTexture_ = copyTexture;
-            }
+            m_source.copyTexture_ = copyTexture ?? CopyTextureHelper.VerticalFlipCopy;
```
[the remark this answers is not in the document]
Comment (aet) [reply Runtime/Scripts/VideoStreamTrack.cs:160]: Forgot there was another constructor CaptureStreamTrack() with CopyTexture textureCopy = null that gets passed around. The original code declared `videoFlip = true` twice, if want to repeat `CopyTexture copyTexture = CopyTextureHelper.VerticalFlipCopy` to both constructors

[the remark this answers is not in the document]
Comment (MaximKurbanov) [reply Runtime/Scripts/VideoStreamTrack.cs:160]: @aet I also wanted to do this
`public VideoStreamTrack(Texture texture, CopyTexture copyTexture = CopyTextureHelper.VerticalFlipCopy)`
 but got Compiler Error CS1736 (Default parameter value for must be a compile-time constant)

[the remark this answers is not in the document]
Comment (MaximKurbanov) [reply Runtime/Scripts/VideoStreamTrack.cs:160]: https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-messages/cs1736?f1url=%3FappId%3Droslyn%26k%3Dk(CS1736)

[the remark this answers is not in the document]
Comment (aet) [reply Runtime/Scripts/VideoStreamTrack.cs:160]: Ah, makes sense. Learn something new every day!

**Comment by karasusan** [comment]
@MaximKurbanov
Thank you for fixing it.
I'm reviewing.

File: Runtime/Scripts/VideoStreamTrack.cs [hunk 2]
```diff
@@ -28,7 +28,7 @@ public class VideoStreamTrack : MediaStreamTrack
         /// <summary>
         /// Flip vertically received video, change it befor start receive video
         /// </summary>
-        public static bool NeedReceivedVideoFlipVertically { get; set; } = false;
+        public static bool NeedReceivedVideoFlipVertically { get; set; } = true;
```
Comment (karasusan) [inline Runtime/Scripts/VideoStreamTrack.cs:31]: I think that we need to set `false` if textures are copied vertically flipped by default.

Comment (MaximKurbanov) [reply Runtime/Scripts/VideoStreamTrack.cs:31]: Before #971 it was true received video flip and sender video needFlip is also true (flip vertically and send, upon receipt of flip back). So "true" here is correct.

Comment (karasusan) [reply Runtime/Scripts/VideoStreamTrack.cs:31]: @karasusan 
Ah, I understood.
I am checking this change with Unity Render Streaming, and it is needed to change like this. It should be no problem.
https://github.com/Unity-Technologies/UnityRenderStreaming/pull/953

**Pull request merged**